### DISCUSSION TITLE
Welcome to vim Discussions!

### DISCUSSION THREAD

**Discussion by k-takata** [vim/vim#12735] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cnrkuo** [comment]
The new chapter begins.

**Comment by NoSuck** [comment]
Thank you for stepping in to fill those giant shoes, @k-takata and friends.  This weekend needed some good news.  Long live Vim.

**Comment by insomnia-creator** [comment]
RIP 🙏

**Comment by Adamkaram** [comment]
RIP

**Comment by dofy** [comment]
RIP 🙏🏾

**Comment by smartcmd** [comment]
RIP

**Comment by petermeng** [comment]
RIP 🙏🏾

**Comment by raveensrk** [comment]
From Normal mode to GOD mode. R.I.P!

**Comment by pendulm** [comment]
RIP, bram, you shine some light in my life.

**Comment by Expector-Hutch** [comment]
R.I.P.

**Comment by AmirHossein-z** [comment]
R.I.P

**Comment by ketancmaheshwari** [comment]
Sorry to hear you left us, Bram. RIP.

**Comment by pheiduck** [comment]
R.I.P Bram, I would invite Parallel Testing, tests will be much faster but more will be flaky maybe.

**Reply by chrisbra** [reply]
parallel testing would be nice, but we already have too many flaky tests and debugging this would even more harder

**Comment by bfrg** [comment]
Currently, Discussions are redirected to `vim_dev`. Wouldn't it be better to have a dedicated mailing list like `vim_discuss` exclusively for GitHub Discussions? I think `vim_dev` should be used only for feature requests and bug reports.

**Reply by chrisbra** [reply]
I hear you, but I think it should be better after a while.

**Comment by andirady** [comment]
R.I.P bram. 🥺

**Comment by ijliym** [comment]
R.I.P bram. 🥺

**Comment by wolandark** [comment]
R.I.P Bram.

What would be the future of vimscript 9?
Will it remain or will it be replaced by another scripting language?
I thought something like this was implied in the google groups discussion.

**Reply by yegappan** [reply]
Vim9 script is stable and has been supported for more than a year.   It will continue to be supported and new features will be added.  There are no plans to change this.

**Reply by wolandark** [reply]
Thank you.

**Comment by jazi007** [comment]
R.I.P Bram 💔
